include/unifex/new_thread_context.hpp

```diff
@@ -65,23 +67,34 @@ class context {
   template <class Receiver>
   friend struct _op;
 
+  class scheduler;
+
   class schedule_sender {
   public:
-    template <template <typename...> class Variant,
-             template <typename...> class Tuple>
+    template <
+        template <typename...>
+        class Variant,
+        template <typename...>
+        class Tuple>
     using value_types = Variant<Tuple<>>;
 
     template <template <typename...> class Variant>
     using error_types = Variant<std::exception_ptr>;
 
     static constexpr bool sends_done = true;
 
-    explicit schedule_sender(context* ctx) noexcept
-      : context_(ctx) {}
+    explicit schedule_sender(context* ctx) noexcept : context_(ctx) {}
 
     template <typename Receiver>
     operation<Receiver> connect(Receiver&& r) const {
-      return operation<Receiver>{context_, (Receiver&&)r};
+      return operation<Receiver>{context_, (Receiver &&) r};
+    }
+
+    template <typename CPO>
+    friend auto tag_invoke(
+        _get_completion_scheduler::_fn<CPO>,
```

Comment:
Is it the `tag_t` causing this to not work, or `get_completion_scheduler`?

Reply:
I've been playing with this and I can't get it to work either. I don't understand enough about the template interactions here to spot what is going wrong. This might  benefit from @ericniebler's greater expertise.

Reply:
The `get_completion_scheduler` CPOs must be `const` or `constexpr`.

Reply:
Hi @ericniebler I have marked get_completion_scheduler to be constexpr variable.... however, still it has the same problem. Am I missing something here?